ext/afform/admin/ang/afGuiEditor/config-form.html

```diff
@@ -18,10 +18,10 @@
 
 <div class="form-group">
   <label for="af_config_form_server_route">
-    {{:: ts('Path:') }}
+    {{:: ts('Page:') }}
   </label>
-  <input ng-model="afform.server_route" class="form-control" id="af_config_form_server_route" />
-  <p class="help-block">{{:: ts('Expose the form as a standalone page on the web site. (Example: "civicrm/my-form")') }}</p>
+  <input ng-model="afform.server_route" class="form-control" id="af_config_form_server_route" pattern="^civicrm\/[-0-9a-zA-z\/_]+$" onfocus="this.value = this.value || 'civicrm/'" onblur="if (this.value === 'civicrm/') this.value = ''" title="{{:: ts('Path must begin with &quot;civicrm/&quot;') }}">
```

Comment:
I think the `pattern` constraint prevents any data from getting out of the model (huzzah). But to get the output styled, I think the key things are from BS3 and AngularJS:

* Bootstrap3 (https://getbootstrap.com/docs/3.4/css/)
    * You should put `has-error` on any `form-group` with validation errors
* AngularJS Validation (https://docs.angularjs.org/guide/forms#custom-validation): 
    * Enforces `pattern` attribute
    * The relevant `[ng-model]` element will automatically get the class `ng-valid` or `ng-invalid`.
    * You can inspect validation status via `FORM_NAME.INPUT_NAME.$error.TYPE` (e.g. `config_form.server_route.$error.pattern`). This is most useful for conditional messaging
        ```html
          <p class="help-block" ng-show="config_form.server_route.$error.pattern">ZOMG</p>
        ```

I pushed up a patch which does a quick fix for this element. So you can see reasonable styling for the error:

<img width="262" alt="Screen Shot 2021-02-03 at 1 16 48 AM" src="https://user-images.githubusercontent.com/1336047/106724793-846ea480-65bd-11eb-92bc-8468458eb7b2.png">

Of course, one would prefer to have a generic rule (erm, `directive`) which says, "Within a Bootstrap3-based screen, watch for `.ng-invalid` and toggle `.has-error` higher-up." There are some examples along those lines in https://stackoverflow.com/questions/14348384/reconcile-angular-js-and-bootstrap-form-validation-styling (*I didn't put any of those example snippets the PR -- something felt a little off, so one should do it more thoughtfully, but they give the gist.*)